docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1013,6 +1015,8 @@ Whether packed row is enabled for YSQL.
 
 Default: `false`
 
+For versions 2.16.4 and later, and 2.18.1 and later, the feature is stable to be used in production, and starting from version 2.19 and later, the flag default is true for new clusters.
```

Comment:
Packed Row for YSQL can be used 2.16.4 release onwards in production environments, if the cluster is not used in xCluster settings. For xCluster scenarios, use 2.18.1 and later. Starting from version 2.19 and later, the flag default is true for new clusters.